<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mouse listener</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Service</string>
    </property>
    <addaction name="actionLoad_device_module"/>
    <addaction name="actionRemove_device_module"/>
    <addaction name="actionStart_mouse_listening"/>
    <addaction name="actionStop_mouse_listening"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_device_module">
   <property name="text">
    <string>Load device kernel module</string>
   </property>
  </action>
  <action name="actionStart_mouse_listening">
   <property name="text">
    <string>Start mouse listening</string>
   </property>
  </action>
  <action name="actionRemove_device_module">
   <property name="text">
    <string>Remove device kernel module</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStop_mouse_listening">
   <property name="text">
    <string>Stop mouse listening</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
